### ISSUE TITLE
dist/index.js not found

### ISSUE THREAD

**Issue by eseiler** [nf-core/setup-nextflow#12]
Hey there,

I'm getting 
```
Error: File not found: '/home/runner/work/_actions/nf-core/setup-nextflow/v1/dist/index.js'
```

with the latest release.

```
      - name: Install Nextflow
        uses: nf-core/setup-nextflow@v1
        with:
          version: latest-stable
```

It seems like the `action.yml` references `dist/index.js`, which does not exist in the latest release (however, a `lib/src/main.js` exists) https://github.com/nf-core/setup-nextflow/blob/v1.3.0/action.yml; but did exist in earlier releases https://github.com/nf-core/setup-nextflow/blob/v1.2.0/action.yml

**Comment by MillironX** [comment]
On it. Not sure how this slipped through the 1.2 release and just showed up now, but I'm working to fix it asap.

**Comment by MillironX** [comment]
Resolved via a manual publish of 7baa5502c9082273913d55071fc9fe06ecdb15af. Gee howdy do I hate automation some days.
